plugins/available/fzf.plugin.bash

```diff
@@ -5,13 +5,15 @@
 cite about-plugin
 about-plugin 'load fzf, if you are using it'
 
-if [ -r ~/.fzf.bash ]; then
-	# shellcheck disable=SC1090
-	source ~/.fzf.bash
-elif [ -r "${XDG_CONFIG_HOME:-$HOME/.config}"/fzf/fzf.bash ]; then
-	# shellcheck disable=SC1091
-	source "${XDG_CONFIG_HOME:-$HOME/.config}"/fzf/fzf.bash
-fi
+if ! _bash-it-component-item-is-enabled plugin blesh; then
```

Comment:
I think `[[ ${BASH_VERSION-} ]]` is visible here because the loading order is supposed to be controlled by `BASH_IT_LOAD_PRIORITY`, so `blesh.plugin.bash` is supposed to be loaded before `fzf.plugin.bash`.

Reply:
Just in case someone decides to change the loading order numbers, Also I didn't see the point in sourcing extra files for no reason. time is money :)

Reply:
I might miss your point, but what are the extra files?